src/Static.jl

```diff
@@ -587,6 +590,10 @@ value is a `StaticInt`.
     end
 end
 
+function Base.invperm(p::Tuple{Vararg{StaticInt,N}}) where {N}
```

Comment:
Doesn't that also define a method for `invperm(p::Tuple{})`?

Reply:
Oh, you're probably right. I'll add another field there to make sure it's not type piracy.